cypress/e2e/yves/claim/claim.cy.ts

```diff
@@ -0,0 +1,313 @@
+import { container } from '@utils';
+import { ClaimListPage, ClaimCreatePage, ClaimDetailPage, OrderDetailsPage, ClaimOrderPage } from '@pages/yves';
+import { ClaimStaticFixtures, ClaimDynamicFixtures } from '@interfaces/yves';
+import { CustomerLoginScenario } from '@scenarios/yves';
+import { CustomerLogoutScenario } from '@scenarios/yves';
+
+(['suite', 'b2b'].includes(Cypress.env('repositoryId')) ? describe : describe.skip)(
+  'claim management',
+  { tags: ['@yves', '@claim'] },
+  (): void => {
+    const claimListPage = container.get(ClaimListPage);
+    const claimCreatePage = container.get(ClaimCreatePage);
+    const claimDetailPage = container.get(ClaimDetailPage);
+    const orderDetailPage = container.get(OrderDetailsPage);
+    const claimOrderPage = container.get(ClaimOrderPage);
+    const customerLoginScenario = container.get(CustomerLoginScenario);
+    const customerLogoutScenario = container.get(CustomerLogoutScenario);
+
+    let staticFixtures: ClaimStaticFixtures;
+    let dynamicFixtures: ClaimDynamicFixtures;
+
+    before((): void => {
+      ({ staticFixtures, dynamicFixtures } = Cypress.env());
+    });
+
+    it('customer should be able to create and view a general claim', (): void => {
+      customerLoginScenario.execute({
+        email: dynamicFixtures.customer.email,
+        password: staticFixtures.defaultPassword,
+        withoutSession: true,
+      });
+
+      claimListPage.visit();
+      claimListPage.clickCreateClaimButton();
+
+      staticFixtures.generalClaim.availableTypes = staticFixtures.claimTypes.general;
+      claimCreatePage.assertPageLocation();
+      claimCreatePage.createClaim(staticFixtures.generalClaim);
+
+      claimDetailPage.assertPageLocation();
+      cy.contains(claimCreatePage.getClaimCreatedMessage()).should('exist');
+
+      var reference: string;
+      cy.url()
+        .then((url) => {
+          const urlParams = new URLSearchParams(url.split('?')[1]);
+          reference = urlParams.get('reference');
```

Comment:
Can we use smth like 

- claimListPage.openLatestClaimDetailsPage();
- laimDetailPage.assertOrderClaimDetails

?